packages/fast-jss-utilities/README.md

```diff
@@ -92,3 +92,23 @@ const styles = {
     }
 };
 ```
+
+### applyAcrylic
+The `applyAcrylic` function can be used to create a partially transparent texture for the background of a specific element or UI surface.

```

Comment:
Do we need to state UI surface? They are all elements.

Reply:
removed